Velaptor/Graphics/FontAtlasMetrics.cs

```diff
@@ -9,37 +9,30 @@ namespace Velaptor.Graphics;
 
 /// <summary>
 /// Provides metric data for a single font atlas texture.
+/// <param name="Rows">Number of rows in the atlas.</param>
+/// <param name="Columns">Number of columns in the atlas.</param>
+/// <param name="Width">Width of the atlas.</param>
+/// <param name="Height">Height of the atlas.</param>
 /// </summary>
-internal struct FontAtlasMetrics
+internal readonly record struct FontAtlasMetrics(uint Rows, uint Columns, uint Width, uint Height)
```

Comment:
I'm getting the [SA1313 warnings](https://duckduckgo.com/?t=h_&q=sa1313+record+struct&ia=web)

Reply:
This is because you are using a primary constructor which is this piece of code:
```internal readonly record struct FontAtlasMetrics(uint Rows, uint Columns, uint Width, uint Height)```

I have plans to convert all kinds of code to primary constructors once dotnet is updated to v8.  
I have to do some research and see what exactly the impact is and this work is planned.

I apologize for this.  I should of thought of this but it slipped my mind!!

Just make the struct like this:
```cs
internal readonly record struct FontAtlasMetrics
{
    /// <summary>
    /// Gets the number of rows in the atlas.
    /// </summary>
    public uint Rows { get; init; }

    /// <summary>
    /// Gets the number of columns in the atlas.
    /// </summary>
    public uint Columns { get; init; }

    /// <summary>
    /// Gets the width of the atlas.
    /// </summary>
    public uint Width { get; init; }

    /// <summary>
    /// Gets the height of the atlas.
    /// </summary>
    public uint Height { get; init; }
}
```

Reply:
Yeah, I was just trying. Just for clarifying. It is suppose to use { get; init} also by default when declared as readonly record so the body would be redundant except by the comments.

Velaptor it's dotnet v7 but the C# reference version is 11, isn't it?, this syntax is supported since C# 10.
![Screenshot_20231119-194113.png](https://github.com/KinsonDigital/Velaptor/assets/104023800/8cbefab8-dcdb-4110-af1a-2360ce93b174)

The code was just like that before my last commit.

Thanks for your comments.

Reply:
Yes, you are correct.  I don't want to choose one thing over another right now because I want to evaluate what direction I want to go.  😀